qemu/tests/cfg/block_resize.cfg

```diff
@@ -4,42 +4,57 @@
     no RHEL.4
     only qcow2 raw
     type = block_resize
-    images += " stg"
-    image_size_stg = 10G
-    image_name_stg = images/stg
-    create_image_stg = yes
-    force_create_image_stg = yes
-    image_snapshot_image1 = yes
-    blk_extra_params_stg = "serial=TARGET_DISK0"
-    Host_RHEL.m6..ide:
-        blk_extra_params_stg = "wwn=0x5000123456789abc"
     block_size_cmd = "fdisk -l | grep {0}"
     block_size_pattern = ",\s+(\d+\s+b)ytes"
     disk_change_ratio = "1.5 0.5"
     guest_prepare_cmd = ""
-    Windows:
-        block_size_cmd = wmic diskdrive get size, index
-        disk_index = 1
-        disk_letter = I
-        format_disk = yes
-        block_size_pattern = "1\s+(\d+)"
-        accept_ratio = 0.005
-        disk_update_cmd = "echo rescan > cmd"
-        disk_update_cmd += " && echo select disk 1 >> cmd"
-        disk_update_cmd += " && echo select partition 1 >> cmd"
-        disk_update_cmd += " && echo extend >> cmd"
-        disk_update_cmd += " && echo exit >> cmd"
-        disk_update_cmd += " && diskpart /s cmd"
-        # "::" is used to separate the commands as it is not used in both Linux and Windows
-        disk_update_cmd += "::"
-        disk_update_cmd += "echo select disk 1 > cmd"
-        disk_update_cmd += " && echo select partition 1 >> cmd"
-        disk_update_cmd += " && echo shrink desired=DISK_CHANGE_SIZE >> cmd"
-        disk_update_cmd += " && echo exit >> cmd && diskpart /s cmd"
-        disk_unit = M
-        disk_rescan_cmd = "echo rescan > cmd"
-        disk_rescan_cmd += " && echo exit >> cmd"
-        disk_rescan_cmd += " && diskpart /s cmd"
+    variants:
+        - with_system_disk:
+            blk_extra_params_image1 = "serial=TARGET_DISK0"
+            backup_image_before_testing = yes
+            restore_image_after_testing = yes
+            Windows:
+                block_size_cmd = wmic diskdrive get size, index
+                disk_index = 1
+                disk_letter = C
+                block_size_pattern = "0\s+(\d+)"
+                accept_ratio = 0.005
+                disk_rescan_cmd = "echo rescan > cmd"
+                disk_rescan_cmd += " && echo exit >> cmd"
+                disk_rescan_cmd += " && diskpart /s cmd"
+        - with_data_disk:
+            images += " stg"
+            image_size_stg = 10G
+            image_name_stg = images/stg
+            create_image_stg = yes
+            force_create_image_stg = yes
+            image_snapshot_image1 = yes
+            blk_extra_params_stg = "serial=TARGET_DISK0"
+            Host_RHEL.m6..ide:
+            blk_extra_params_stg = "wwn=0x5000123456789abc"
```

Comment:
From line 14, not sure is it the original cfg file which you mentioned.
    Host_RHEL.m6..ide:
         blk_extra_params_stg = "wwn=0x5000123456789abc"